Website/src/rendering/src/Layout.tsx

```diff
@@ -35,12 +35,19 @@ const Layout = ({ context }: LayoutProps): JSX.Element => {
     context.pageState === LayoutServicePageState.Preview
       ? 'experience-editor-active'
       : '';
+  const contextTitle = context['EventInfo'] as NodeJS.Dict<string | string>;
+  let pageTitle = contextTitle.titlePrefix;
+  if (route?.fields?.pageTitle?.value) {
+    pageTitle += ' - ' + route?.fields?.pageTitle?.value;
+  } else {
+    pageTitle += ' - Page';
+  }
   // END CUSTOMIZATION
 
   return (
     <>
       <Head>
-        <title>{route?.fields?.pageTitle?.value || 'Page'}</title>
+        <title>{pageTitle}</title>
```

Comment:
Please add a DEMO TEAM CUSTOMIZATION comment block around this line for easier future upgrades